README.md

```diff
@@ -84,149 +94,106 @@ const UserProfile = ({ userId }) => (
+### `createSingleEntryResource`
 
-1. If there is no data for this resource OR no entry in the cache for the arguments (See [invalidateAfter](https://github.com/chadwatson/react-remote-resource/blob/master/README.md#createremoteresource)) OR the cache is stale (See [invalidateAfter](https://github.com/chadwatson/react-remote-resource/blob/master/README.md#createremoteresource)), then the `load` function of the `categoriesResource` and `productsResource` will be invoked and promises thrown.
-2. If either of the promises reject, the closest `RemoteResourceBoundary` will handle the error. If both promises resolve, the `categoriesById` and `products` data will be available to use (as the first item in the tuple).
-3. You can set the state using the second item in the tuple. Resource state changes, unlike component based `useState`, will persist in memory. If a component unmounts and remounts the state will be the same as when you left it.
+An opinionated version of `createResource` that assumes there is only one entry in the resource state. The getter function simply returns the last data that was fetched. The setter function simply sets the resource state to the data that was fetched. And the predicate function simply checks if the resource state is not `undefined`. Basically the loader function will only be called once if it successfully resolves.
 
-### `useResourceActions`
+```javascript
+const myResource = createSingleEntryResource(authToken =>
+  fetch(`/api/about_me?auth_token=${authToken}`)
+);
+```
 
-A React hook that takes a resource and an optional array of arguments and returns an object literal with the following methods:
+### `createKeyedResource`
 
-- `set`: A function that takes either the new state to set for this entry or a function that takes the current state of the entry and returns what should be set as the new state for the entry.
+An opinionated version of `createResource` that stores retrieved data in an object literal, allowing you to create a key for each entry. The getter function returns the data that is associated with key that your key creating function returns. The setter function sets the resource state to the data that was fetched. And the predicate function simply checks if the resource state is not `undefined`.
 
-- `refresh`: A function that allows you to bypass the check against the `updatedAt` timestamp and immediately refetch the data. _Note: the promise will **not** be thrown for this action._
+```javascript
+const myResource = createKeyedResource(
+  // A function that takes all of the arguments that are supplied to the loader and uses the returned value as the key
+  (authToken, userId) => userId,
+  (authToken, userId) => fetch(`/api/users/${userId}?auth_token=${authToken}`)
+);
+```
 
-- `save`: The `save` function that was defined with `createRemoteResource`. _Note: this will be `undefined` if you did not define a `save` function with `createRemoteResource`._
+### `createTimedSingleEntryResource`
```

Comment:
**Requested Change**

This ideally would be directly under `createSingleEntryResource`

Reply:
By "directly under" do you mean hierarchically or just the next section after `createSingleEntryResource`.

So, like this?
```markdown
### `createSingleEntryResource`
...

#### `createTimedSingleEntryResource`
...
```

or like this?

```markdown
### `createSingleEntryResource`
...

### `createTimedSingleEntryResource`
...
```

Reply:
Sorry! The next section! Mainly the context is easier to see when they are close!

Reply:
👍